appveyor.yml

```diff
@@ -6,25 +6,27 @@ install:
   - cd C:\projects\
   - if not exist dependencies2013.zip curl -kLO https://obsproject.com/downloads/dependencies2013.zip -f --retry 5 -C -
   - 7z x dependencies2013.zip -odependencies2013
-  - if not exist qt570.zip curl -kLO https://www.slepin.fr/obs-websocket/ci/qt570.zip -f --retry 5 -C -
-  - 7z x qt570.zip -o"Qt5.7.0"
   - set DepsPath32=%CD%\dependencies2013\win32
   - set DepsPath64=%CD%\dependencies2013\win64
-  - set QTDIR32=%CD%\Qt5.7.0\msvc2013
-  - set QTDIR64=%CD%\Qt5.7.0\msvc2013_64
+  - set QTDIR32=C:\Qt\5.7\msvc2013
+  - set QTDIR64=C:\Qt\5.7\msvc2013_64
   - set build_config=Release
   - git clone --recursive https://github.com/jp9000/obs-studio
```

Comment:
If the `lib`s are stored is there a need to checkout `obs-studio` and run cmake?

Reply:
I'm not entirely sure.  The CMake file for obs-websocket uses LIBOBS_INCLUDE_DIR.  I figured that if it wasn't available and specified that CMake might fail.  I could always test it later, but I was trying to make minimal changes at first.